src/libsystemd-network/sd-dhcp-lease.c

```diff
@@ -780,6 +843,37 @@ int sd_dhcp_lease_load(sd_dhcp_lease **ret, const char *lease_file) {
                            "ROUTES", &routes,
                            "CLIENTID", &client_id_hex,
                            "VENDOR_SPECIFIC", &vendor_specific_hex,
+                           "OPTION_224", &options[DHCP_OPTION_PRIVATE_0 - DHCP_OPTION_PRIVATE_START],
```

Comment:
So DHCP_OPTION_PRIVATE_0 - DHCP_OPTION_PRIVATE_START is fancy way to say 0. Why not just say
```
options,
options + 1,
options + 2,
```
?